Vector.sc

```diff
@@ -82,6 +92,10 @@ AbstractVector[slot] : ArrayedCollection {
 		}
 	}
 
+	isVector {
```

Comment:
I believe this is not needed. isKindOf can be used.

Reply:
Got it; removed. Did it as a new commit instead of reverting the old once since I know of at least one clone out there of my own fork that was made in between submitting this and editing.